cmd/common/oci.go

```diff
@@ -327,9 +377,42 @@ func NewRunCommand(rootCmd *cobra.Command, runtime runtime.Runtime, hiddenColumn
 		AddOCIFlags(cmd, operatorParams, skipParams, runtime)
 	}
 
+	cmd.PersistentFlags().StringVarP(&inFile, "file", "f", "", "path to gadget runtime configuration to apply")
```

Comment:
This flag is available for `ig`, but it's not working. Mmmm in general, should it be there?

Reply:
It was not correctly initializing the operators in that case, I fixed it.